### DISCUSSION TITLE
When does Dependabot rebase PRs?

### DISCUSSION THREAD

**Discussion by stayko-chalakov** [dependabot/dependabot-core#9463] [Q&A]
Hi there, I have switched on dependabot in one of my GitHub repositories (private). I have a dependabot config file which has a schedule to create PRs on a Monday morning. I have about 5 PRs opened from Dependabot but they are all behind as I was pushing other features in the mean time. 

They haven't been rebased since yesterday so wanted to know when and how often does Dependabot rebase them? Or what triggers the rebasing? 

I noticed in other repos that if I merge one Dependabot PR it will trigger a rebase for all of them but in this case I didn't merge any of the initial batch.. so they are all stale. 

I'm working on an auto merge script so was wondering..

Thanks

**Comment by gkorland** [comment]
Did you find the answer?
